src/Chapter2.hs

```diff
@@ -336,7 +348,9 @@ from it!
 ghci> :l src/Chapter2.hs
 -}
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
+subList _ _ [] = []
+subList x 0 y = []
+subList x y b = take (y-x+1) $ drop x b
```

Comment:
Great solution!
Note that in Haskell all operators/functions are separated from its arguments with spaces 👾 

```suggestion
subList x y b = take (y - x + 1) $ drop x b
```

Reply:
Hmmmm.. hahahah understood